old/industry.csv

```diff
@@ -195,4 +195,5 @@ Daniel Wigdor,University of Toronto,http://www.dgp.toronto.edu/~dwigdor,x3hvdTsA
 Frank Dellaert,Georgia Institute of Technology,https://www.cc.gatech.edu/~dellaert,ZxXBaswAAAAJ
 Sanja Fidler,University of Toronto,http://www.cs.utoronto.ca/~fidler,CUlqK5EAAAAJ
 Jyh-Ming Lien,George Mason University,http://cs.gmu.edu/~jmlien,EnrEVb0AAAAJ,iRobot
+Xin Xia 0001,Huawei,https://xin-xia.github.io,XSZRxOEAAAAJ
```

Comment:
Please follow the format of other entries - keep the original affiliation and put the new affiliation at the end.

Reply:
Done. Thank you very much！